README.md

```diff
@@ -1 +1,61 @@
-# J2ZROMERO.github.io
\ No newline at end of file
+# Personal_Portafolio_Web_Site![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portafolio
+
+> You could find more information about my experience, projects , achieves , career path and much more just take a look at this content.
+
+
+## Built With
+
+- HTML
+- CSS
+- FLEX BOX
+- Linters
+ 
+## Getting Started
+
+**While you are navigating on my portafolio you could download,test and edit each project you choose and you like just read the instructions of each project**
+
+
+
```

Comment:
- [ ] Please add your link of deployment in the Readme.md file so it would be easily accessible to all the other people :blush: